features/sign_in.feature

```diff
@@ -32,3 +32,11 @@ Feature: Sign in
     And   I should be at the page I was on
     And   I should receive a "Signed out successfully." notification
 
+  Scenario: Sign out after using Money Manager
```

Comment:
@picandocodigo tbh this scenario can be moved to another file as it is a new feature. 

```
#file: features/enhance_signout_behaviour.feature
.
.
.
```